consensus/src/round_manager.rs

```diff
@@ -1007,22 +1111,67 @@ impl RoundManager {
                         BlockStage::QC_AGGREGATED,
                     );
                 }
-                self.new_qc_aggregated(qc, vote.author()).await
+                QC_AGGREGATED_FROM_VOTES.inc();
+                self.new_qc_aggregated(qc.clone(), vote.author())
+                    .await
+                    .context(format!(
+                        "[RoundManager] Unable to process the created QC {:?}",
+                        qc
+                    ))?;
+                if self.onchain_config.order_vote_enabled() {
```

Comment:
depend on the flag, we need to move the fast share broadcast too

Reply:
When order votes are enabled, fast shares are broadcasted together with order votes right?
Should we piggyback order votes with fast shares into the same message? Or should we broadcast them separately at the same time?

Reply:
Separately is better. They are controlled by different flags.

Reply:
Done.